Module Import Error

I discovered a bug that occurs with pytest when this template is used to create a new repository whose name is one word with no hyphens, such as `oneword`. In these instances, pytest fails to identify the submodules present and fails to import modules needed to test.

Error output:

```
Run pytest -v -x -n auto
  pytest -v -x -n auto
  shell: /usr/bin/bash -e {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.10.[2](https://github.com/viperior/oneword/runs/5612093769?check_suite_focus=true#step:6:2)/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/[3](https://github.com/viperior/oneword/runs/5612093769?check_suite_focus=true#step:6:3).10.2/x6[4](https://github.com/viperior/oneword/runs/5612093769?check_suite_focus=true#step:6:4)/lib
============================= test session starts ==============================
platform linux -- Python 3.10.2, pytest-7.1.1, pluggy-1.0.0
rootdir: /home/runner/work/oneword/oneword, configfile: pytest.ini, testpaths: tests
plugins: xdist-2.[5](https://github.com/viperior/oneword/runs/5612093769?check_suite_focus=true#step:6:5).0, forked-1.4.0
gw0 I / gw1 I

==================================== ERRORS ====================================
______________ ERROR collecting tests/test_flip_coins_function.py ______________
ImportError while importing test module '/home/runner/work/oneword/oneword/tests/test_flip_coins_function.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/hostedtoolcache/Python/3.10.2/x[6](https://github.com/viperior/oneword/runs/5612093769?check_suite_focus=true#step:6:6)4/lib/python3.[10](https://github.com/viperior/oneword/runs/5612093769?check_suite_focus=true#step:6:10)/importlib/__init__.py:[12](https://github.com/viperior/oneword/runs/5612093769?check_suite_focus=true#step:6:12)6: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_flip_coins_function.py:7: in <module>
    from games_of_chance.flip_coins import flip_coins
E   ModuleNotFoundError: No module named 'games_of_chance'
=========================== short test summary info ============================
ERROR tests/test_flip_coins_function.py
!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!! xdist.dsession.Interrupted: stopping after 1 failures !!!!!!!!!!!!!
=============================== 1 error in 0.44s ===============================
Error: Process completed with exit code 2.
```

This should be repairable by prefixing `python -m` to the "quick" version of the pytest call.

Before:

```
pytest -v -x -n auto
```

After:

```
python -m pytest -v -x -n auto
```

--- a/.github/workflows/python-app.yml
+++ b/.github/workflows/python-app.yml
@@ -29,7 +29,7 @@ jobs:
         flake8 . --count --max-complexity=10 --max-line-length=100 --statistics
     - name: Test with pytest (quick)
       run: |
-        pytest -v -x -n auto
+        python -m pytest -v -x -n auto
     - name: Test with pytest
       run: |
         python -m coverage run -m pytest -v

